### PR TITLE
HTS Training fixes

### PR DESCRIPTION
**Pull request by brian-mw**

### CODE REVIEW

File: app/services/hts_service/ait_intergration/contact_csv_row_builder.rb [hunk 1]
```diff
@@ -10,7 +10,15 @@ def last_name contact, index
     end
 
     def sex contact, index
-      contact['Gender of contact']
+      case contact['Gender of contact']&.strip
```
Comment (fchiyenda) [inline app/services/hts_service/ait_intergration/contact_csv_row_builder.rb:13]: For the gender of the contact do always assumme they are not pregnant?

File: app/services/hts_service/ait_intergration/contact_csv_row_builder.rb [hunk 2]
```diff
@@ -10,7 +10,15 @@ def last_name contact, index
     end
 
     def sex contact, index
```
Comment (fchiyenda) [inline app/services/hts_service/ait_intergration/contact_csv_row_builder.rb]: Why are we maintaining the index argument

Comment (brian-mw) [reply app/services/hts_service/ait_intergration/contact_csv_row_builder.rb:12]: it is an index as in contact[0], contact[1]..   not the index client (naming issue)

File: app/services/hts_service/ait_intergration/contact_csv_row_builder.rb (hunk 1 above)
Comment (brian-mw) [reply app/services/hts_service/ait_intergration/contact_csv_row_builder.rb:13]: resolved

File: app/services/hts_service/ait_intergration/contact_csv_row_builder.rb [hunk 3]
```diff
@@ -1,87 +1,85 @@
...
       contact['Age of contact'].to_i * 12
     end
 
-    def age contact, index
+    def age contact
       contact['Age of contact'].to_i
     end
 
-    def age_group contact, index
+    def age_group contact
```
Comment (fchiyenda) [inline app/services/hts_service/ait_intergration/contact_csv_row_builder.rb:82]: These age groups might change at some point don't for hard code the we can revise in the future.

File: app/services/hts_service/ait_intergration/contact_csv_row_builder.rb [hunk 4]
```diff
@@ -95,117 +93,113 @@ def age_group contact, index
...
       csv_row_builder.health_facility_id contact
     end
 
-    def health_facility_name contact, index
+    def health_facility_name contact
       csv_row_builder.health_facility_name contact
     end
 
-    def district_id contact, index
+    def district_id contact
```
Comment (fchiyenda) [inline app/services/hts_service/ait_intergration/contact_csv_row_builder.rb:120]: This should return the district location_id the hash currently returning an integer

**Review by fchiyenda — CHANGES_REQUESTED** [review]
District id hash update should be also include in the index

**Review by fchiyenda — APPROVED** [review]

**Pull request merged**